File: GEOEmergency_Final/DTO/EmergencyImageDTO.cs
using Microsoft.AspNetCore.Http;
using System.ComponentModel.DataAnnotations;
using GEOEmergency.API.Models;

namespace GEOEmergency.DTOs
{
    public class EmergencyImageDTO
    {
        [Required]
        public string Description { get; set; }

        [Required]
        public IFormFile Image { get; set; }
        
        [Required]
        public TargetDepartment TargetDepartment { get; set; }
        
        public double? Latitude { get; set; }
        public double? Longitude { get; set; }
        public string? Address { get; set; }
    }
}

File: emergency/src/main/java/com/emergency/dto/EmergencyImageDTO.java
// EmergencyImageDTO.java
package com.emergency.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.*;
import org.springframework.web.multipart.MultipartFile;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class EmergencyImageDTO {
    private Long citizenId;
    @NotBlank
    private String description;
    private MultipartFile image;
    private String targetDepartment = "BOTH";
    private Double latitude;
    private Double longitude;
    private String address;
}
